barretenberg/cpp/src/barretenberg/dsl/acir_format/ecdsa_secp256k1.cpp

```diff
@@ -25,30 +25,6 @@ secp256k1_ct::g1_ct ecdsa_convert_inputs(Builder* ctx, const bb::secp256k1::g1::
     return { x, y };
 }
 
-// vector of bytes here, assumes that the witness indices point to a field element which can be represented
```

Comment:
This function is also used in ecdsa_secp265r1, and my PR lead to a linker error. I tried to explicitly instantiate but I was getting weird errors so I just gave up and moved this function to the header.

Reply:
hm weird